After Uninstalling & Installing of latest CKEdiitor is crashing on PB > Settings > Site Settings > Site Behavior (tab) > More (sub tab) > Open HTML Editor Manager

## Description & Steps to reproduce
Uninstall CKeditor and then installed again with CKEditor package in Dnn.platform 
- 9.3.0
- 9.2.2

## Current Behavior
PB > Settings > Site Settings > Site Behavior (tab) > More (sub tab) > Open HTML Editor Manager is crashing

## Expected Behavior
It should load the settings without any error

## Logs
Error in logs: 
2019-02-21 08:36:53.597+05:00 [DESKTOP-GJON2V9][D:20][T:171][FATAL] DotNetNuke.Framework.PageBase - An error has occurred while loading page. 
System.NullReferenceException: Object reference not set to an instance of an object. 
   at DotNetNuke.Modules.HtmlEditorManager.Presenters.ProviderConfigurationPresenter.LoadCurrentEditor(String editorName) 
   at DotNetNuke.Modules.HtmlEditorManager.Presenters.ProviderConfigurationPresenter.View_Initialize(Object sender, EventArgs e) 
   at System.EventHandler.Invoke(Object sender, EventArgs e) 
   at DotNetNuke.Web.Mvp.ModuleViewBase.PageInitComplete(Object sender, EventArgs e) 
   at System.EventHandler.Invoke(Object sender, EventArgs e) 
   at System.Web.UI.Page.OnInitComplete(EventArgs e) 
   at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint) 
2019-02-21 08:36:53.625+05:00 [DESKTOP-GJON2V9][D:20][T:171][ERROR] DotNetNuke.Services.Exceptions.Exceptions - ~/Default.aspx?tabid=32&error=Object+reference+not+set+to+an+instance+of+an+object.&content=0 
System.NullReferenceException: Object reference not set to an instance of an object. 
   at DotNetNuke.Modules.HtmlEditorManager.Presenters.ProviderConfigurationPresenter.LoadCurrentEditor(String editorName) 
   at DotNetNuke.Modules.HtmlEditorManager.Presenters.ProviderConfigurationPresenter.View_Initialize(Object sender, EventArgs e) 
   at System.EventHandler.Invoke(Object sender, EventArgs e) 
   at DotNetNuke.Web.Mvp.ModuleViewBase.PageInitComplete(Object sender, EventArgs e) 
   at System.EventHandler.Invoke(Object sender, EventArgs e) 
   at System.Web.UI.Page.OnInitComplete(EventArgs e) 
   at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint) 


## Snapshot

![installing_error](https://user-images.githubusercontent.com/43787378/53364969-d5050e00-3961-11e9-92b3-5802979d97cc.png)

## Affected Versions
- 9.3.0
- 9.2.2

--- a/DNNConnect.CKEditorProvider.dnn
+++ b/DNNConnect.CKEditorProvider.dnn
@@ -84,7 +84,7 @@
             <install>
               <configuration>
                 <nodes>
-                  <node path="/configuration/dotnetnuke/htmlEditor[@defaultProvider='']" action="updateattribute" name="defaultProvider" value="DNNConnect.CKE" />
+                  <node path="/configuration/dotnetnuke/htmlEditor[@defaultProvider]" action="updateattribute" name="defaultProvider" value="DNNConnect.CKE" />
                   <node path="/configuration/dotnetnuke/htmlEditor/providers" action="update" key="name" collision="ignore">
                     <add name="DNNConnect.CKE" type="DNNConnect.CKEditorProvider.CKHtmlEditorProvider, DNNConnect.CKEditorProvider" providerPath="~/Providers/HtmlEditorProviders/DNNConnect.CKE/" settingsControlPath="~/Providers/HtmlEditorProviders/DNNConnect.CKE/Module/EditorConfigManager.ascx" />
                   </node>

--- a/DNNConnect.CKEditorProvider.dnn
+++ b/DNNConnect.CKEditorProvider.dnn
@@ -84,7 +84,7 @@
             <install>
               <configuration>
                 <nodes>
-                  <node path="/configuration/dotnetnuke/htmlEditor[@defaultProvider]" action="updateattribute" name="defaultProvider" value="DNNConnect.CKE" />
+                  <node path="/configuration/dotnetnuke/htmlEditor[@defaultProvider='' or @defaultProvider='DotNetNuke.RadEditorProvider']" action="updateattribute" name="defaultProvider" value="DNNConnect.CKE" />
                   <node path="/configuration/dotnetnuke/htmlEditor/providers" action="update" key="name" collision="ignore">
                     <add name="DNNConnect.CKE" type="DNNConnect.CKEditorProvider.CKHtmlEditorProvider, DNNConnect.CKEditorProvider" providerPath="~/Providers/HtmlEditorProviders/DNNConnect.CKE/" settingsControlPath="~/Providers/HtmlEditorProviders/DNNConnect.CKE/Module/EditorConfigManager.ascx" />
                   </node>

